cartridge/roles/metrics.lua

```diff
@@ -87,26 +82,23 @@ local function validate_config(conf_new)
     return validate_routes(conf_new.export)
 end
 
--- table to store enabled routes
-local current_paths = {}
-
 local function apply_routes(export)
     local httpd = cartridge.service_get('httpd')
     if httpd == nil then
         return
     end
     for _, exporter in ipairs(export) do
         local path, format = remove_side_slashes(exporter.path), exporter.format
-        if current_paths[path] ~= format then
-            if current_paths[path] then
+        if metrics_vars.current_paths[path] ~= format then
+            if metrics_vars.current_paths[path] then
                 delete_route(httpd, path)
             end
             httpd:route({method = 'GET', name = path, path = path}, handlers[format])
-            current_paths[path] = format
+            metrics_vars.current_paths[path] = format
         end
     end
```

Comment:
I'd suggest to refactor this function logic:

1. Remove all unwanted routes
```lua
for path, current_format in pairs(metrics_vars.current_paths) do
    if export[path] ~= current_format then
        -- remove the route
    end
end
```

2. Set up all missing routes